services/transcribe.py

```
@@ -126,7 +127,39 @@ def download_and_transcribe(self, video_url: str, clip_id: str) -> tuple[str, Pa
+            'format': 'best[ext=mp4]/best',  # Prefer MP4 format
+            'quiet': True,
+            'no_warnings': True,
+        }
+        
+        try:
+            with yt_dlp.YoutubeDL(ydl_opts) as ydl:
+                ydl.download([url])
+            
+            # Verify the file was downloaded
+            if not output_path.exists():
+                raise RuntimeError("yt-dlp download failed - file not found")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)